Vendoring error while building

**Describe the bug**
```
erson@erson-69:~/Workspace/Go/skywire$ make build
mkdir -p ./apps
GO111MODULE=on go build "-ldflags=-X github.com/skycoin/dmsg/buildinfo.version=v0.6.0-9-g6f691a14 -X github.com/skycoin/dmsg/buildinfo.date=2022-03-02T14:38:37Z -X github.com/skycoin/dmsg/buildinfo.commit=6f691a14e32fe7cd1b5a97ca0c8b3cbece69adf0 -X github.com/skycoin/skywire/pkg/visor.BuildTag=" -mod=vendor  -o ./apps/ ./cmd/apps/skychat
go: inconsistent vendoring in /home/erson/Workspace/Go/skywire:
        github.com/jaypipes/ghw@v0.8.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt
        github.com/ghodss/yaml@v1.0.0: is explicitly required in go.mod, but not marked as explicit in vendor/modules.txt

        To ignore the vendor directory, use -mod=readonly or -mod=mod.
        To sync the vendor directory, run:
                go mod vendor
make: *** [Makefile:143: host-apps] Error 1
```

--- a/go.mod
+++ b/go.mod
@@ -53,7 +53,6 @@ require (
 require (
 	github.com/go-chi/chi/v5 v5.0.8-0.20220103230436-7dbe9a0bd10f
 	github.com/james-barrow/golang-ipc v0.0.0-20210227130457-95e7cc81f5e2
-	github.com/jaypipes/ghw v0.8.0
 )
 
 require (
@@ -68,7 +67,6 @@ require (
 	github.com/getlantern/context v0.0.0-20190109183933-c447772a6520 // indirect
 	github.com/getlantern/errors v1.0.1 // indirect
 	github.com/getlantern/hex v0.0.0-20190417191902-c6586a6fe0b7 // indirect
-	github.com/ghodss/yaml v1.0.0 // indirect
 	github.com/go-chi/chi v4.1.2+incompatible // indirect
 	github.com/go-ole/go-ole v1.2.4 // indirect
 	github.com/go-stack/stack v1.8.0 // indirect

